gateway/src/apicast/policy/standalone/standalone.lua

```diff
@@ -319,13 +351,27 @@ end
 local function find_service(self, route)
   local destination = route and route.destination
 
-  if self.services and destination then
+  if not destination then return nil, 'no destination' end
+
+  if destination.service then
     return self.services[destination.service]
   end
+
+  if destination.upstream then
+    local upstream = self.upstreams[destination.upstream]
+
+    if upstream then
+      return upstream
```

Comment:
also return here upstream, nil no?

Reply:
Return `value` and `value, nil` work exactly the same. So I guess there is not much reason for that.